app/main.py

```diff
@@ -1,6 +1,27 @@
+import requests
+
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = "3e243be0a1204288bfc132613231304"
```

Comment:
what? Never ever put such sensitive data in such an insecure manner as you've done here. Use `.env`.
This is a very huge mistake